### PR TITLE
Hide chart controls (disaggregations) when for map/embed tabs

### PR DESCRIPTION
**Pull request by Norric1Admin**
@SavvasStephanides 
https://github.com/ONSdigital/sdg-indicators/pull/2993/

### CODE REVIEW

**Comment by Norric1Admin** [comment]
@brockfanning This update hides the dissagregation and unit options (see [test](https://sustainabledevelopment-cyprus.github.io/3-a-1/)). Please can this be added to v.0.4.0?

File: _layouts/indicator.html [hunk 1]
```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
   </div>
 </div>
 
-{% include footer.html %}
+<style type="text/css">
```
Comment (brockfanning) [inline _layouts/indicator.html]: Could this be added in the [Sass file](https://github.com/open-sdg/open-sdg/blob/master/assets/css/default.scss)?

File: _layouts/indicator.html [hunk 2]
```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
...
+  #chartControls[show="1"]{display: block;}
+  #chartControls[show="0"]{display: none;}
+</style>
 
+<script>
```
Comment (brockfanning) [inline _layouts/indicator.html]: Could this be added in the [indicator view file](https://github.com/open-sdg/open-sdg/blob/master/_includes/assets/js/indicatorView.js)?

File: _layouts/indicator.html [hunk 3]
```diff
@@ -45,17 +45,19 @@ <h2>{{ meta.graph_title | t }}</h2>
...
     <div class="col-md-3">
-      <div id="toolbar">
-        <span id="series-help">
-          <h4>{{ t.indicator.sub_categories }}</h4>
-          <p>{{ t.indicator.series_help }}</p>
-        </span>
-        <p>{{ t.indicator.legend_help }}</p>
-        <div id="fields">
-          <p class="async-loading" data-img="{{ site.baseurl }}/assets/img/loading-small.gif"></p>
+      <div id="chartControls" show="1">
```
Comment (brockfanning) [inline _layouts/indicator.html]: I'm not a huge HTML-validation nitpicker, but just to be thorough I think this should be `data-show` instead of `show`, right?

File: _layouts/indicator.html [hunk 4]
```diff
@@ -73,17 +75,17 @@ <h4>{{ t.indicator.sub_categories }}</h4>
         <div class="async-loaded" style="display:none">
           <ul class="nav nav-tabs data-view" role="tablist">
             <li role="presentation" class="nav-item active">
-              <a class="nav-link" data-toggle="tab" href="#chartview" aria-controls="chartview" role="tab">{{ t.indicator.chart }}</a>
+              <a class="nav-link" data-toggle="tab" href="#chartview" onClick='ga("send", "event", "Tab change", "Change data view", "Change to Chart tab")' aria-controls="chartview" role="tab" show-controls="1">{{ t.indicator.chart }}</a>
```
Comment (brockfanning) [inline _layouts/indicator.html]: We'll want to remove the ga() stuff, since this ticket is not related to Google Analytics.

And as above probably `data-show-controls` instead of `show-controls`.

File: _layouts/indicator.html [hunk 5]
```diff
@@ -171,5 +173,29 @@ <h4>{{ t.indicator.sub_categories }}</h4>
   </div>
 </div>
 
-{% include footer.html %}
+<style type="text/css">
+  #chartControls[show="1"]{display: block;}
```
Comment (brockfanning) [inline _layouts/indicator.html]: Do you think it might be possible to get a little fancy here, and to "grey out" the toolbar instead of hiding it altogether? I think that `pointer-events:none` would effectively disable it, and maybe an "overlay" div or something to grey out the contents? On desktop it just looks a little unexpected to have that chunk of the screen just be white.

**Comment by Norric1Admin** [comment]
Don't merge this yet - it is incompatible with Internet Explorer: https://github.com/ONSdigital/sdg-indicators/pull/3036

**Comment by brockfanning** [comment]
@SavvasStephanides @Norric1Admin Just checking in on this one. As you can see above I had made a few comments that I think needed addressing, plus the IE issue. I'm assuming we still want this feature right?

**Comment by Norric1Admin** [comment]
Thanks for your comments @brockfanning - we'll review this again in June/July.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning @Norric1Admin can this be merged in?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin It has conflicts at the moment, but also there were a few things I thought might be improved (see comments above).

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning so I've just tried to recreate this with [this branch](https://github.com/open-sdg/open-sdg/blob/remove-disagg-map-embed) with the suggestions you made and it hasn't seemed to work, [looking here](https://sdg.mango-solutions.com/remove-disaggs/3-a-1/)

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin The logic in this approach is tough for me to follow, so I gave this a shot. My goal was to keep it as simple as possible. I also gave it a little animation to make it clear what's happening. See #233

**Comment by LucyGwilliamAdmin** [comment]
Closing this as new PR has been made (#233)
